app/views/layouts/generic.slim

```diff
@@ -0,0 +1,20 @@
+doctype html
+html lang="#{@language_code.try(:downcase)}"
+  head
+    title  = @title
+    meta name="viewport" content="width=device-width, initial-scale=1.0"
+    meta name="description" content=t('branding.description')
+    = stylesheet_link_tag "sumofus"
```

Comment:
with extracted assets we need to be careful about what assets we include. You'd need to make this template generic and have another with the sumofus stylesheet stored in the assets repos.